client/me/purchases/manage-purchase/plan-details/billing-period.jsx

```diff
@@ -51,12 +51,14 @@ class PlanBillingPeriod extends Component {
 		if ( isRenewing( purchase ) && purchase.renewMoment ) {
 			return translate( 'Billed yearly, renews on %s', {
 				args: purchase.renewMoment.format( 'LL' ),
+				comment: '%s the renewal date in format M DD, Y, for example: June 10, 2019',
```

Comment:
`%s` could be the subject of a well-formed sentence:
```
%s is the renewal date in format M DD, Y, e.g., June 10, 2019
```

Reply:
🤦‍♂ 

That's what I was trying to do actually. Thanks, fixing now 😄

Reply:
Fixed in e60c036874528d1acc1b67f2ed30ec6a5ebb5318